### ISSUE TITLE
Failure on android (kotlin's version)

### ISSUE THREAD

**Issue by Katekko** [aarajput/wc_flutter_share#3]
```
Launching lib/main.dart on LM X210 in debug mode...
* Error running Gradle:
ProcessException: Process "/Users/joaofelipe/Desktop/APPS/dacasapay-cliente-flutter/android/gradlew" exited abnormally:
Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :app
WARNING: API 'variant.getMergeAssets()' is obsolete and has been replaced with 'variant.getMergeAssetsProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variant.getMergeAssets(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.
WARNING: API 'variantOutput.getProcessResources()' is obsolete and has been replaced with 'variantOutput.getProcessResourcesProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variantOutput.getProcessResources(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.

> Configure project :wc_flutter_share
WARNING: API 'variant.getJavaCompile()' is obsolete and has been replaced with 'variant.getJavaCompileProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variant.getJavaCompile(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.
         *********************************************************
WARNING: This version of camera will break your Android build if it or its dependencies aren't compatible with AndroidX.
         See https://goo.gl/CP92wY for more information on the problem and how to fix it.
         This warning prints for all Android build failures. The real root cause of the error may be unrelated.
         *********************************************************


FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.3.0 and higher.
The following dependencies do not satisfy the required version:
project ':wc_flutter_share' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.71

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 30s
  Command: /Users/joaofelipe/Desktop/APPS/dacasapay-cliente-flutter/android/gradlew app:properties

Please review your Gradle project setup in the android/ folder.
Exited (sigterm)
```

What can I do now?

**Comment by Katekko** [comment]
Can you upgrade the kotlin's version to at least 1.3?
![image](https://user-images.githubusercontent.com/13632735/60999958-c6c4f680-a332-11e9-9762-d73064e19f82.png)

**Comment by Katekko** [comment]
Approve my PR to fix

https://github.com/aarajput/wc_flutter_share/pull/4

**Comment by aarajput** [comment]
Hey,
Thanks for submitting PR.
I have approved and released it with version 0.1.1.
